spec/rubocop/cop/naming/constant_name_spec.rb

```diff
@@ -40,6 +40,10 @@
     expect_no_offenses('TOP_TEST, TEST_2 = 5, 6')
   end
 
+  it 'allows screaming snake case with accents' do
+    expect_no_offenses('TÖP_TEST = 5')
+  end
+
```

Comment:
[IMO] We should have a test case. The test case tests a constant with small case character with accents.

For example:

```ruby
it 'registers an offense for ...' do
  expect_offense(<<-RUBY)
    Aí_BCDEF = 5
    ^^^^^^^^^^ message
  RUBY
end
```